content/en/docs/releases/bugs/index.md

```diff
@@ -63,18 +73,38 @@ containing:
     $ kubectl --namespace istio-system get cm -o yaml
     {{< /text >}}
 
-* Current and previous logs from all Istio components and sidecar
+* Current and previous logs from all Istio components and sidecars:
 
-* Istiod logs:
+  Here some examples on how to obtain those, please adapt for your environment:
```

Comment:
Indentation is not correct in this area. See https://deploy-preview-10307--preliminary-istio.netlify.app/latest/docs/releases/bugs/.

Reply:
You can run `make serve` locally to start a server for your current docs and then modify the docs and see the output after a short delay. This allows one to figure out how changes might affect the output.

Indention can be a tricky thing and using `make serve` can usually help one figure it out fairly quickly. 

If you need some help, let me know (easiest is probably on the Istio Slack) and I can help you through the steps or spend a few minutes getting it to look correct.

Reply:
I improved it but could make it look how I wanted. Please take a look and let me know if it is ok like this now.

Reply:
When I first looked the page in the current deploy, I'm a little thrown of by the two bullets:

<img width="790" alt="Screen Shot 2021-09-09 at 3 08 44 PM" src="https://user-images.githubusercontent.com/10537847/132755475-095703dc-9ed9-4486-9d4a-f88231711ec9.png">

I'm wondering if they should be at the same indentation level. It seems that the first one shouldn't be bulleted, or the set from istiod logs to all artifacts might need to be indented once more. Does this make seem better?

<img width="933" alt="Screen Shot 2021-09-09 at 3 08 20 PM" src="https://user-images.githubusercontent.com/10537847/132755546-d60aa665-4578-41ff-95c5-bd542baa42c5.png">

Reply:
Yes, to have bullet point for `Current and previous logs...` and then have istiod, ingres logs, egress logs etc under that. That was my original idea but looked completely wrong as you noted in https://github.com/istio/istio.io/pull/10307#discussion_r705617949

The way printed now, it seems `Current and prevous logs...` is under configmaps, but it should not be...

But I don't have strong preferences. Would like me to update as this last picture you shared?

Reply:
Ah, never mind this. I played a bit more with the syntax and got it to output how I wanted. Please take a look at the new commit. Thanks,